examples/step-27/step-27.cc

```diff
@@ -186,7 +186,7 @@ namespace Step27
 
   template <int dim>
   LaplaceProblem<dim>::LaplaceProblem()
-    : dof_handler(triangulation)
+    : dof_handler(triangulation, true)
```

Comment:
Maybe be a bit more specific
```suggestion
    : dof_handler(triangulation, /*enable_hp_capability */ true)
```